libs/plugins/scully-plugin-playwright/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { findPlugin, green, log, logError, logOk, registerPlugin, routeRenderer } from '@scullyio/scully';
+import { exec } from 'child_process';
+import { LaunchOptions } from 'playwright';
+import { playwrightRender } from './lib/plugins-scully-plugin-playwright';
+import { launchedBrowser, launchedBrowser$ } from './lib/plugins-scully-plugin-playwright-utils';
+
+async function runScript(cmd: string) {
+  return new Promise((resolve, reject) => {
+    exec(cmd, (err, stdout, stderr) => {
+      if (err) {
+        log(stderr);
+        reject(err);
+      } else {
+        resolve(stdout);
+      }
+    });
+  });
+}
+const plugin = async () => {
```

Comment:
The install of the scripts should go into the schematics, or oackage.json, not from code?